src/main/org/deidentifier/arx/ARXResult.java

```diff
@@ -383,6 +386,21 @@ public DataHandle getOutput(boolean fork) {
         if (optimalNode == null) { return null; }
         return getOutput(optimalNode, fork);
     }
+    
+    /**
+     * Returns a score
+     *  
+     * @param node the transformation
+     * @param metric the metric
+     * 
+     * @return
+     */
+    public double getScore(ARXNode node, Metric<?> metric) {
```

Comment:
This is obviously not the right place for introducing this method. Please leave the class ARXResult untouched.